src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -518,6 +524,23 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   }
 
   /**
+    *  Publish workflow structure to centralized logging.
+    */
+  private def publishWorkflowEvent() : Unit = {
+    import org.json4s._
+    import org.json4s.native.Serialization.{write}
+
+    logger.info(s"--------> publishWorkflowEvent ")
+    implicit val formats = DefaultFormats + FieldSerializer[WorkflowExecution]()
+    implicit val actorSystem = context.system
+
+    for {
+      status <- globalDataAccess.getWorkflowExecution(workflow.id)
+    } yield PubSubMediator.publish(WorkflowExecutionEvent, write(status))
```

Comment:
Does this mean to use the `PubSubMediator` object for publishing?  The `workflowLogger` is created above and handles the subscription but is never used again.